client-v2/src/selectors/collectionSelectors.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import { createSelector } from 'reselect';
```

Comment:
I've never come across selectors before, this is very cool.

Reply:
Yes, I know! @RichieAHB introduced me to them.